src/pybel/io/extras.py

```diff
@@ -20,24 +21,45 @@
 ]
 
 
-def to_graphml(graph: BELGraph, path: Union[str, BinaryIO]) -> None:
```

Comment:
The problem with adding more graphml exporters is the user probably won't know what to pick. Maybe if we want several different schemata for graphml, we should have a second argument called `schema` that delegates to several other functions

```python
def to_graphml(graph, schema=None):
    if schema is None or schema == 'simple':
        return _to_graphml_simple(graph)
    elif schema == 'belframework-counterintuitive':
        return _to_graphml_counterintuitive(graph)
    else:
        raise ValueError('Unhandled schema: {}'.format(schema))
```

Reply:
Resolved in a2686ca18e03c98aaa5446afc38f940d25ea6487